web-server/views/index.ejs

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Green Lean Electrics</title>
+    <%- include('./partials/bootstrap-css') %>
+    <link rel="stylesheet" type="text/css" href="/css/index-gle.css">
+  </head>
+  <body>
+    <div id="headerBackground"></div>
+    <div id="main">
+      <%- include('./partials/appbar', {
+          title: 'Green Lean Electrics',
+          logo: 'logo-gle',
+          nav: {Home: '/', Prosumer: '/prosumer/signin', Manager: '/manager/signin'},
+          navActive: 'Home'
+          }) %>
+      <div id="header">
+        <div id="headerContent">
+          <h3>Monitor the electricity grid</h3>
+          <strong>or</strong>
+          <h3>Manage the electricity grid</h3>
+        </div>
+      </div>
+      <div id="content">
+          <h1>Hello</h1><br>
```

Comment:
Shoud there be 13 hello here?

Reply:
No this was just to see the background parallax effect but I didn't know what to put there. I will remove this.